src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -3001,4 +3001,30 @@
       </Comments>
     </Property>
   </Node>
+  <Node Name="IWithExpressionOperation" Base="IOperation" ChildrenOrder="Value,Initializer">
```

Comment:
>IWithExpressionOperation [](start = 14, length = 24)

I think we would want a special rewrite for this node in CFG, something very similar to a an object initializer operation